ccdproc/core.py

```diff
@@ -165,6 +165,10 @@ def ccd_process(ccd, oscan=None, trim=None, error=False, master_bias=None,
         If True, scale the dark frame by the exposure times.
         Default is ``False``.
 
+    gain_corrected : bool, optional
+        If True, the calibration frames have already been gain corrected.
```

Comment:
Maybe a hint which parameters correspond to the "calibration frames" could be added.